packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.base.tsx

```diff
@@ -42,6 +42,15 @@ export class ChoiceGroupBase extends BaseComponent<IChoiceGroupProps, IChoiceGro
     this._labelId = getId('ChoiceGroupLabel');
   }
 
+  /**
+   * Gets the current checked option.
+   */
+  public get optionChecked(): IChoiceGroupOption | undefined {
```

Comment:
>optionChecked [](start = 13, length = 13)

Do you think should it be named as `checkedOption`? Current name seems more like a `boolean` whether the option is checked or not. Again, it's up to you.

Reply:
Will change the name as you have a point with the semantics. I believe the reason I went with `optionChecked` is that the state properties are `keyChecked` and `keyFocused` ))